migrations/versions/570_create_brief_responses_table.py

```diff
@@ -0,0 +1,34 @@
+"""Create brief responses table
+
+Revision ID: 570
+Revises: 560
+Create Date: 2016-02-10 12:19:22.888832
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '570'
+down_revision = '560'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    op.create_table('brief_responses',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('data', postgresql.JSON(), nullable=True),
```

Comment:
I can't think how/why we would have a brief response with NULL data? Seems odd to allow "the response is no response".